homeworkdop.py

```diff
@@ -0,0 +1,85 @@
+"""Additional homework
+Maksim Murashko
+Automatic python tester course"""
+
+# 1. Однажды ночью вы отправляетесь кататься на мотоцикле.
+# В 00:00 вы запускаете двигатель,
+# и встроенный таймер автоматически начинает
+# отсчитывать продолжительность вашей поездки в минутах.
+# Вы отправляетесь исследовать окрестности.
+# Когда вы, наконец, решаете вернуться назад,
+# вы понимаете, что есть вероятность,
+# что мосты на вашем пути домой разведены,
+# и вы останетесь в затруднительном положении! К сожалению,
+# у вас нет с собой часов и вы не знаете, который час.
+# Благодаря таймеру мотоцикла вы знаете только то,
+# что с 00:00 прошло n минут.
+# Используя таймер мотоцикла,
+# рассчитайте текущее время.
+# Возвращает ответ в виде суммы цифр,
+# которую покажет цифровой таймер в формате чч:мм.
+
+print("Задание №1.")
+t1 = int(input("Введите колличество минут: "))
+a = t1 // 60
+b = t1 % 60
+print(f"Результат {a:02}:{b:02}")
+
+# 2 Вы играете в ролевую игру.
+# В настоящее время общее количество
+# ваших очков опыта (XP) равно опыту.
+# Чтобы достичь следующего уровня,
+# ваш XP должен быть как минимум на пороге.
+# Если вы убьете монстра перед собой,
+# вы получите больше очков опыта в размере награды.
+#
+# Учитывая значения опыта,
+# и награды, проверьте,
+# достигнете ли вы следующего уровня после
+# убийства монстра.
+#
+# Пример:
+# Для experience = 10, threshold = 15, и reward = 5,
+# результат должен быть: true
+# Для experience = 10, threshold = 15, и reward = 4,
+# результат должен быть: false
+
+EXPERIENCE = 10
+THRESHOLD = 15
+REWARD = [5, 4]
+for i in REWARD:
+    EXPERIENCE = 10
+    EXPERIENCE = EXPERIENCE + i
+    if EXPERIENCE >= THRESHOLD:
+        print("Задание №2:")
+        print(f'experience = 10, threshold = 15, и reward = {i} -', True)
+    else:
+        print(f'experience = 10, threshold = 15, и reward = {i} -', False)
+
+# 3 Учитывая целые числа a и b,
+# определите, приводит ли следующий псевдокод к бесконечному циклу.
+#
+# пока a не равно b, делай
+#    увеличить а на 1
+#    уменьшить b на 1
+# Предположим, что программа выполняется на виртуальной машине,
+# которая может хранить произвольные длинные числа и выполняться вечно.
+#
+# Пример:
+# Для a = 2 и b = 6, результат должен быть: false
+# Для a = 2 и b = 3, результат должен быть: true
+
```

Comment:
Please use the following numbers to check:
1) 0, 1 -> true
2) 10, 10 -> false
3) 5, 10 -> true
4) 6, 10 -> false
5) 10, 0 -> true
6) 5, 5 -> false